### PR TITLE
Add Accept Offer modal to Offer pages

### PR DESCRIPTION
**Pull request by delventhalz**
Adds an Accept Offer Modal, which also allows for Holding creation. Also reverse the direction of the bifold arrow in most contexts. The direction of the arrow no reflects the direction of the flow of assets, relative to the user.

_Screenshots:_
screen shot 2017-12-20 at 1 17 32 pm

### CODE REVIEW

**Comment by boydjohnson** [comment]
When the AcceptOffer modal opens I get in the console log: TypeError: holdings.find(...) is undefined.

File: sawbuck_app/src/views/accept_offer_modal.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        if (offer.error) return console.error(offer.error)
+        vnode.state.offer = offer
+
+        return Promise.all([
+          api.get(`accounts/${api.getPublicKey()}`),
+          api.get(`accounts/${offer.owners[0]}`)
+        ])
+      })
+      .then(([ user, owner ]) => {
+        const inAsset = getAsset(vnode.state.offer.source, owner.holdings)
```
Comment (peterschwarz) [inline sawbuck_app/src/views/accept_offer_modal.js:154]: Based on feedback from @boydjohnson, is there a condition where the holdings is null?

Comment (delventhalz) [reply sawbuck_app/src/views/accept_offer_modal.js:154]: Shouldn't be. Accounts should be sent back with `"holdings": []` if they have no holdings.

**Review by boydjohnson — APPROVED** [review]
It would be nice if instead of "New Holding" it said "New Sawbuck Holding" or "New USD Holding".

**Comment by boydjohnson** [comment]
Tested locally and was able to accept offers.

**Review by peterschwarz — APPROVED** [review]

**Pull request merged**